backend/src/main/java/com/morak/back/poll/domain/PollItem.java

```diff
@@ -42,21 +42,34 @@ public PollItem(Long id, Poll poll, String subject) {
         this.subject = subject;
     }
 
-    public void addPollResult(Member member) {
-        PollResult pollResult = new PollResult(null, this, member);
+    public void addPollResult(Member member, String description) {
+        PollResult pollResult = new PollResult(null, this, member, description);
         pollResults.add(pollResult);
     }
 
-    public void deleteIfPollMember(Member member) {
+    public void deletePollResultIfPollMember(Member member) {
         pollResults.removeIf(pollResult -> pollResult.getMember().equals(member));
     }
 
-    public List<Member> getMembersByAnonymous() {
+    public List<PollResult> getResultsByAnonymous() {
         if (poll.getIsAnonymous()) {
-            return new ArrayList<>();
+            return pollResults.stream()
+                    .map(PollResult::fromAnonymous)
+                    .collect(Collectors.toList());
         }
+        return pollResults;
+    }
+
+    public Boolean isSelectedBy(Member member) {
+        return pollResults.stream()
+                .anyMatch(result -> result.isSameMember(member));
+    }
+
+    public String getDescriptionFrom(Member member) {
         return pollResults.stream()
-                .map(PollResult::getMember)
-                .collect(Collectors.toList());
+                .filter(result -> result.isSameMember(member))
+                .findFirst()
+                .orElseGet(PollResult::new)
+                .getDescription();
```

Comment:
null 문자열을 얻기 위함인 것 같은데, 
```suggestion
                .orElseGet(null);
```
```suggestion
                .orElseGet("");
```

는 어떤가요 ?

Reply:
null 보다는 "" 추천